account_payment_order_notification/models/account_payment_order.py

```diff
@@ -36,7 +36,10 @@ def action_view_notifications(self):
 
     def _action_send_mail_notifications(self, template):
         for notification in self.notification_ids:
-            notification.message_post_with_source(template)
+            notification.message_post_with_source(
+                template,
+                email_layout_xmlid="mail.mail_notification_light",
```

Comment:
Why is it necessary to specify `email_layout_xmlid` now? I don't think it should be necessary.